nar_solidfire_sds_install/README.md

```diff
@@ -68,21 +68,6 @@ Example Playbook
            - /dev/disk/by-id/<id for sdc>
 ```
 
-Testing the role (Molecule testing)
```

Comment:
Hi Thomas, these have been removed from the code for the posting we are doing this Wed, the 14th.